.github/workflows/test-package.yml

```diff
@@ -3,9 +3,9 @@ name: Dart CI
 on:
   # Run on PRs and pushes to the default branch.
   push:
-    branches: [ master ]
+    branches: [ dart-dev, master ]
```

Comment:
Is this on purpose? Should we only have `[ master ]` there until we start using the stable branch and then this should be `[ stable, master]`?

Reply:
Okay sure

Reply:
Updated to only master for now.